views/components/BarGraphTooltip/index.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import { TOTAL_LABEL } from '../../../constants'
-import { siteTooltipContent } from '../../fe-utils/siteTooltipContent'
+import { siteTooltipContent } from './helpers'
+import { isTargetShown } from './helpers'
```

Comment:
we don't need both of these imports

Reply:
good catch https://github.com/AMP-SCZ/dpdash/pull/210/commits/0c891fa440840faf5410fba6bac78747b91f6088